samples/expo/withSampleSentry.ts

```diff
@@ -0,0 +1,153 @@
+import fs from 'fs';
+import type { ConfigPlugin, XcodeProject } from 'expo/config-plugins';
+import { withAppBuildGradle, withDangerousMod, withXcodeProject } from 'expo/config-plugins';
+
+const SENTRY_GRADLE_LOCAL_OVERWRITE = `../../../../sentry.gradle`;
```

Comment:
This is pointing outside of the root folder.
```suggestion
const SENTRY_GRADLE_LOCAL_OVERWRITE = `../../../sentry.gradle`;
```

Reply:
The file is at `/samples/expo/android/app/build.gradle`.

I believe `../../../../sentry.gradle` is correct.